lsp-bridge.el

```diff
@@ -80,10 +80,6 @@
 (require 'markdown-mode)
 (require 'diff)
 
-(defvar acm-library-path (expand-file-name "acm" (if load-file-name
-                                                     (file-name-directory load-file-name)
-                                                   default-directory)))
-(add-to-list 'load-path acm-library-path t)
```

Comment:
I don't understand why remove those code? 

I think that making such modifications will increase the cost of using lsp-bridge for users.

Reply:
Oh it's because byte compilation won't automatically pick this up, so I thought it would be better to be explicit all the time. Do you you think it would be better to leave in though?

Reply:
Oh yep it probably won't, but the main reason I wanted to get this working is that the Nix packages expect the files to be byte compiled, and as nice bonus it also acts as a lint check.

I think that native compilation will make more a difference (and that's also part of the Nix build process if you have it enabled), and it'll also need an explicit load path for acm.

Reply:
Users can manually load acm into the load-path, but I don't think this code should be removed. Otherwise, all existing lsp-bridge users would be unable to use lsp-bridge after the update, and adding some redundancy would not cause the byte-compilation process to fail.